src/components/Whiteboard/ElementBehaviors/Resizable/ResizeHandle.tsx

```diff
@@ -148,26 +151,27 @@ export type DragEvent = {
 };
 
 export type ResizeHandleProps = {
+  handleProperties: HandleProperties;
```

Comment:
At least to me it looks a bit strange that `ResizeHandleProps` has a property for the handle.
Are not all of these then handle properties?

Reply:
Good catch 🙂 I had the same thought and somehow sticked to that generic term. But it is basically all about the handle positioning (name and coords). I will rename it.

Reply:
Looks better now, thanks :)